### ISSUE TITLE
PHP message: PHP Warning:  count(): Parameter must be an array or an object that implements Countable in /var/www/html/shop/wp-content/plugins/cloudflare/src/WordPress/Hooks.php on line 133"

### ISSUE THREAD

**Issue by enoch85** [cloudflare/Cloudflare-WordPress#223]
Hi!

As the support page for this plugin on Wordpress doesn't seem maintained, I turned here instead.

I'm seeing this in my error logs:
```
PHP message: PHP Warning:  count(): Parameter must be an array or an object that implements Countable in /var/www/html/shop/wp-content/plugins/cloudflare/src/WordPress/Hooks.php on line 133"
```
Which isn't that pleasant as it seems to happen during checkout in my shop.

It's been 10 months since this plugin was updated, and I've seen many reports on the Wordpress support page which haven't been answered. Do you plan to solve this in the near future?

Thanks!

**Comment by maxfenton** [comment]
Same issue.

**Comment by walkafwalka** [comment]
You should not be displaying PHP errors and warnings. You should look into getting them hidden unless you are on a development server.

**Comment by AiratHalitov** [comment]
Fixed in #231
